webapp/channels/src/components/admin_console/system_users/system_users.tsx

```diff
@@ -78,8 +69,12 @@ function SystemUsers(props: Props) {
                 direction: tableOptions?.direction,
                 ...getSortColumnForOptions(tableOptions?.sortColumn),
                 ...getSortDirectionForOptions(tableOptions?.sortIsDescending),
+                search_term: tableOptions?.searchTerm,
             };
 
+            const {data: count} = await props.getUserCountForReporting(options);
+            setUserCount(count);
+
```

Comment:
this would make the count and reports sequential which is something we want to avoid, the above usEffect which got removed can still be there and it could take in the changed dependencies for total. We can extract the function to get the userReport options into a seperate function which can be used in get user reports and for total reports.